pkg/tnf/handlers/bootconfig/bootconfig.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package bootconfig
+
+import (
+	"time"
+
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+)
+
+const (
+	successfulOutputRegex = `(?s).+`
```

Comment:
One limitation of using an expect based process to control the PTY is the fact that we shouldn't trigger `ReelMatch(...)` based on empty (nil) output. The reason for this is that, if you match the empty string, `ReelMatch(...)` is triggered regardless of whether command execution has completed, which can lead to some nasty and inconsistent race conditions.

As of now, we do not mandate this requirement in code, but we should. Can you instead change to a regular expression that expects some output?

Additionally, I will create a story to address checking that a given reel.Handler does not contain an reel.Step.Expect regular expression string that matches the empty String. Unfortunately, since Go does not support constructors (or inheritance really well for that matter), this will need to be a runtime check at the time of the test Run; better than nothing though.